sdk/search/search-documents/package.json

```diff
@@ -140,6 +140,7 @@
     ],
     "requiredResources": {
       "Azure Search Documents instance": "https://docs.microsoft.com/azure/search/search-create-service-portal"
-    }
+    },
+    "skipFolder": true
```

Comment:
skip doesn't seem to handle export const variables well. Below is the pipeline run where "export const" still errored out despite the file being added to skip.

Pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1108822&view=logs&j=8a044361-f13b-57ee-1142-5d5c4f6a8fd2&t=69a37eb1-ba53-5cbe-d4c5-97ba05de502b&l=412
Corresponding commit: https://github.com/Azure/azure-sdk-for-js/commit/2ab35480a4f4f20008c07ce26735bba514326363